packages/expo-notifications/ios/EXNotifications/PushToken/EXPushTokenModule.m

```diff
@@ -20,6 +20,9 @@ @interface EXPushTokenModule ()
 @property (nonatomic, strong) EXPromiseResolveBlock getDevicePushTokenResolver;
 @property (nonatomic, strong) EXPromiseRejectBlock getDevicePushTokenRejecter;
 
+@property (nonatomic, strong) EXPromiseResolveBlock unregisterForNotificationsResolver;
+@property (nonatomic, strong) EXPromiseRejectBlock unregisterForNotificationsRejecter;
+
```

Comment:
Looks like it's no longer used 😉 

```suggestion
```

Reply:
Good catch! I forgot to clean this up after changing my approach.